src/vs/workbench/contrib/preferences/browser/settingsEditor2.ts

```diff
@@ -1131,21 +1131,27 @@ export class SettingsEditor2 extends EditorPane {
 		return match && match[1];
 	}
 
+	private hasSome(set: Set<string>): boolean {
```

Comment:
I don't think we need a method for this, and the type says `set` is non-null, so you don't need the first part of the check